apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx

```diff
@@ -49,12 +50,14 @@ const WebinarListItem = ({ webinar }: WebinarListItemProps) => {
           )}
         </Typography>
       )}
-      <Typography variant='h6'>{webinar.title}</Typography>
-      <Stack mt={2} direction='row' alignItems='center' color='primary.main'>
-        <LinkButton
-          href={`/webinars/${webinar.slug}`}
-          label={t('webinar.goToWebinar')}
-        />
+      <Stack justifyContent='space-between' flexGrow={1}>
+        <Typography variant='h6'>{webinar.title}</Typography>
+        <Stack mt={2} direction='row' alignItems='center' color='primary.main'>
```

Comment:
Is this `Stack` necessary?

Reply:
Yes, without this we do not achieve the desired behavior

Reply:
I think that you don't need to use a `Stack` if it has only one child. A `Box` is enough

Reply:
I misunderstood which `<Stack/>` you were referring to, so I proceed to update the `<Stack/>` on line 55